### PR TITLE
Removed duplicate RegisterServlet and modified register.jsp

### PR DESCRIPTION
**Pull request by UnitingDust**
- Removed duplicate RegisterServlet lines in web.xml, which was causing errors when pushing code to Google's AppEngine site. 

- Modified register.jsp so that users can see the "About" tab when on /register

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/register.jsp [hunk 1]
```diff
@@ -19,7 +19,8 @@
      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
    <% } else{ %>
      <a href="/login">Login</a>
-   <% } %>
+   <% } %>   
+   <a href="/about.jsp">About</a>
```
Comment (ryanluo) [inline src/main/webapp/WEB-INF/view/register.jsp]: Why did you add this change? In general we want to separate bug fixes and feature implementation so we can easily rollback.

Comment (UnitingDust) [reply src/main/webapp/WEB-INF/view/register.jsp]: Removed.
